release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md

```diff
@@ -0,0 +1,17 @@
+## 2022-08-11 Version 2.2.0.0
+
+Compatible with Opensearch-Dashboards 2.2.0
+
+### Test Fix
+
+* Add tests for account-nav-button when multitenancy is disabled ([#1020](https://github.com/opensearch-project/security-dashboards-plugin/pull/1020))
+
+### Bug Fix
+
+* Revert "Replace _opendistro route with _plugins ([#895](https://github.com/opensearch-project/security-dashboards-plugin/pull/895))" ([#1035](https://github.com/opensearch-project/security-dashboards-plugin/pull/1035))
```

Comment:
Could we rewrite this so its more human readable?

Reply:
Changed this to "Fix bug in SAML support after renaming". Should we edit the name of the PR as well?

Reply:
Users/admins read release notes, developers read PRs, its ok to leave it as it was IMO